examples/slate-demo.ipynb

```diff
@@ -0,0 +1,354 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Imports for firefly_client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from firefly_client import FireflyClient"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import firefly_slate_demo as fs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example tentatively uses localhost:8080 as the server\n",
+    "\n",
+    "'slate.html' is a template made for grid view "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "host='localhost:8080'\n",
+    "channel = 'myChannel8'\n",
+    "html = 'slate.html'\n",
+    "\n",
+    "fc = FireflyClient(host, channel=channel, html_file=html)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dispaly tables, images, XY charts, and Histograms in Window/Grid like layout\n",
+    "\n",
+    "Each rendering unit on Firefly Slate Viewer is called a'cell'. To render a cell and its content, the cell location (row, column, width, height) and a cell ID are needed.  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We use astropy here for convenience, but firefly_client itself does not depend on astropy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import astropy.utils.data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Please refer to firefly_slate_demo.py to get functions which are made to render cells with the elements which could be tables, images, xy charts or histograms onto Firefly Slate Viewer. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Display tables and catalogs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Open a browser to the firefly server in a new tab. Only works when running the notebook locally."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "fc.launch_browser()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Add some tables into cell 'main' (default grid viewer id for tables)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# first table in cell 'main' \n",
+    "fs.load_moving_table(0,0,4,2, fc)"
```

Comment:
For readability, here and below in the notebook, please call out the arguments explicitly.  row=0, col=0, width=4, height=2, fc=fc.

A Markdown cell explaining what width and height mean would be helpful. I mean, width and height are clearly in terms of some kind of cell size. Is the cell size fixed by the Firefly server? Is it configurable?